tests/remote/register/test_register.py

```diff
@@ -70,6 +70,13 @@ def test_create_dockerfile(tmp_path, mocker, local_installable_module):
 # update image
 RUN apt update -y
 
+# create a non-root user
+RUN addgroup --gid 1001 group
+RUN adduser --disabled-password --gecos "" --uid 1001 --gid 1001 --home /home/user user
+ENV PYTHONPATH /home/user
+WORKDIR /home/user
+USER user
+
```

Comment:
Can we somehow factorize some of this and reuse part of the `DOCKERFILE_TEMPLATE`?

Reply:
This is part of the test. We are checking that the dockerfile template is what we expect it to be. 
Else, we would just be checking that Python f-strings are working as intended, and that should be tested in the standard library :)

Reply:
I meant, I think we could avoid updating twice the constant part (where we just run commands that does not depend on anything else) and extract it in a dedicated variable that could be re-injected in the expected result (precisely because we can assume that (f-)strings work as intended) so the test actually focus on the templating part.